tests/integration/components/categories-list-test.js

```diff
@@ -30,11 +41,11 @@ test('it renders the categories and sorts them by name', function(assert) {
   ];
 
   this.set('categories', categories);
-  this.render(hbs`{{categories-list categories=categories}}`);
+  page.render(hbs`{{categories-list categories=categories}}`);
 
-  assert.equal(this.$('.categories-list').length, 1);
-  assert.equal(this.$('.category-item').length, 3);
-  assert.equal(this.$('.category-item:eq(0)').text().trim(), 'Alphabets');
-  assert.equal(this.$('.category-item:eq(1)').text().trim(), 'Society');
-  assert.equal(this.$('.category-item:eq(2)').text().trim(), 'Zoology');
+  assert.ok(page.isRendered);
```

Comment:
I think the way the other selectors work they can't work if the elements are hidden, but I could be wrong. The main point of those tests was not really to check for visibility, but as a smoke test.